qiskit/compiler/assemble.py

```diff
@@ -111,7 +111,7 @@ def assemble(experiments,
         memory_slot_size (int):
             Size of each memory slot if the output is Level 0.
 
-        rep_time (int): repetition time of the experiment in μs.
+        rep_time (float): repetition time of the experiment in s.
```

Comment:
👍 Yes, this is right. We do have to also convert this input from seconds to microseconds before the Qobj is built, within the assemble function. This is how it was done for the frequency arguments (from Hz input to GHz assembled Qobj): https://github.com/Qiskit/qiskit-terra/pull/3597

Reply:
This PR should also show you what kind of changes you might have to make to the test cases. Let me know if it makes sense!:)